taskfile.yaml

```
@@ -66,4 +65,19 @@ tasks:
     cmds:
       - goose -dir migrations sqlite3 subscriptions.db status
 
+  test:unit:
+    desc: "Run unit tests"
+    cmds:
+      - go test ./...
```

```suggestion
      - go test -v ./...
```

This will show each test's name and output in the console